### PR TITLE
Upon importing isambard the code to check for DSSP would freeze 

### LINKED ISSUE #22: DSSP waits for key to be pressed
On windows DSSP waits for a key to be pressed if called without arguments. 

when importing isambard in ipython the whole program is blocked. 

Adding --version when calling DSSP should fix this. 

I can do a pull request.

### PR DESCRIPTION
**Pull request by ajasja**
(waiting for any key press).

Adding --version  to the command being called fixes the problem
Closes #22.

### CODE REVIEW

**Comment by ChrisWellsWood** [comment]
Can you merge into a new branch instead? Master is protected so that it won't allow a merge unless CircleCI passes first.

File: isambard/external_programs/dssp.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ def check_dssp_avail():
     is_dssp_available = False
     if os.path.isfile(global_settings['dssp']['path']):
         try:
-            subprocess.check_output([global_settings['dssp']['path']], stderr=subprocess.DEVNULL)
+            subprocess.check_output([global_settings['dssp']['path'],'--version'], stderr=subprocess.DEVNULL)
```
Comment (ChrisWellsWood) [inline isambard/external_programs/dssp.py]: Please use spaces between list items.

**Comment by ajasja** [comment]
Hmm, I don't think I can merge into a new branch
https://stackoverflow.com/questions/9796370/github-pull-request-to-a-new-upstream-branch

**Comment by ChrisWellsWood** [comment]
That's annoying, I might make a dev branch in future, and you can fork from that.

**Comment by ChrisWellsWood** [comment]
This pull request actually broke ISAMBARDs DSSP integration, as previously we were *expecting* a `CalledProcessError` if everything was working. This has since been rectified in commit 23b61ed.

**Comment by ajasja** [comment]
Sorry! I did run the test suite (but I only got some errors, that I thought
were strictly windows related).

**Comment by ChrisWellsWood** [comment]
It's no problem, this was obscured by CircleCI failing due to a helpful upgrade they made, that's why we didn't catch it sooner. All good now though! 😃

**Pull request merged**